kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/InitService.java

```diff
@@ -33,24 +27,12 @@
 @ApplicationScoped
 public class InitService extends KogitoService {
 
-	@Inject
-	private BaseEntityUtils beUtils;
-
-	@Inject
-	private UserToken userToken;
-
-	@Inject
-	private QwandaUtils qwandaUtils;
-
-	@Inject
-	CacheManager cacheManager;
-
 	@Inject
 	Logger log;
 
-	private static List<List<Attribute>> batchedAttributesList = new LinkedList<>();
+	private static Map<String, List<List<Attribute>>> batchedAttributesListPerProduct = new HashMap<>();
```

Comment:
Concerns static variables that caused the form issue before. We need to find alternative ways to replace or check carefully before merging